partner_relations/view/res_partner_relation.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="form_res_partner_relation" model="ir.ui.view">
+            <field name="model">res.partner.relation</field>
+            <field type="xml" name="arch">
+                <form string="Partner Relation">
+                    <sheet>
+                        <field name="left_partner_id" />
+                        <field name="type_id" />
+                        <field name="right_partner_id" />
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <record id="tree_res_partner_relation" model="ir.ui.view">
+            <field name="model">res.partner.relation</field>
+            <field name="arch" type="xml">
+                <tree
+                    string="Partner Relations"
+                    colors="gray:date_end and date_end &lt; current_date or not active;blue:date_start &gt; current_date"
+                    editable="bottom"
```

Comment:
IMHO, this should be ```editable="top"```, because if there are lots of relations, when user clicks to create a new one, apparently nothing happends, user must go to the end of the screen to see an editable line.